packages/api/src/controllers/userFarmController.js

```diff
@@ -587,25 +575,20 @@ const userFarmController = {
           .first()
           .select('*');
         res.status(201).send(userFarm);
+        try {
+          const { farm_name } = userFarm;
+          const user = await userModel.getUserByEmail(email);
+          await emailModel.createTokenSendEmail(
+            { email, gender, birth_year, language: user ? user.language_preference : language },
```

Comment:
`language_preference` should be used here since emailModel tries to access the property name of `language_preference` rather than `language`